### PR TITLE
Recipe fixes

### PR DESCRIPTION
**Pull request by DbpGaming**
just some recipe fixes, to make the mod play way more like GT4, commits should have self explanatory names, also includes a jack hammer fix e told me to do, any of these are open to discussion on removing them, changing them whatever

"make standard casings allow steel" is a mistype, it actually allows aluminium

### CODE REVIEW

File: src/main/java/gtc_expansion/tile/multi/GTCXTileMultiIndustrialBlastFurnace.java [hunk 1]
```diff
@@ -226,19 +227,19 @@ public static void init() {
...
 		/* Nichrome **/
-		addRecipe(new IRecipeInput[]{metal("Nickel", 4), metal("Chrome", 1)}, 2000, COST_HIGH, GTMaterialGen.getHotIngot(GTCXMaterial.Nichrome, 5));
-		addRecipe(new IRecipeInput[]{input("dustNichrome", 1)}, 2000, COST_HIGH, GTMaterialGen.getHotIngot(GTCXMaterial.Nichrome, 1));
+		addRecipe(new IRecipeInput[]{metal("Nickel", 4), metal("Chrome", 1)}, 2000, COST_HIGH*6, GTMaterialGen.getHotIngot(GTCXMaterial.Nichrome, 5));
+		addRecipe(new IRecipeInput[]{input("dustNichrome", 1)}, 2000, COST_HIGH*2, GTMaterialGen.getHotIngot(GTCXMaterial.Nichrome, 1));
 
 		/* Aluminium **/
 		IRecipeInput aluminium = new RecipeInputCombined(1, new RecipeInputOreDict("dustAluminum"), new RecipeInputOreDict("dustAluminium"));
-		addRecipe(new IRecipeInput[]{aluminium}, 1000, COST_MED, GTMaterialGen.getIngot(GTMaterial.Aluminium, 1));
+		addRecipe(new IRecipeInput[]{aluminium}, 1700, COST_HIGH, GTMaterialGen.getIngot(GTMaterial.Aluminium, 1));
```
Comment (Trinsdar) [inline src/main/java/gtc_expansion/tile/multi/GTCXTileMultiIndustrialBlastFurnace.java]: change the heat back to 1000, I purposely left it at 1000 cause it just seemed dumb to me to need such high heat for such a soft metal

File: src/main/java/gtc_expansion/material/GTCXMaterial.java [hunk 2]
```diff
@@ -116,7 +117,7 @@
     public static final GTMaterial Tin = new GTMaterial("Tin", 220, 220, 220, molten, smalldust, nugget, plate, stick, gear);
     public static final GTMaterial TungstenSteel = new GTMaterial(-1, "Tungstensteel", 4, 100, 100, 160, false, molten, smalldust, dust, nugget, ingot, hotIngot, plate, stick, gear, blockMetal, hull, pipeFluid);
     public static final GTMaterial Uvarovite = new GTMaterial("Uvarovite", 180, 255, 180, dustAll);
-    public static final GTMaterial Zinc = new GTMaterial(30,"Zinc", 1, 250, 240, 240, molten, smalldust, dust, nugget, ingot, plate, stick, gear, blockMetal, tinydust);
+    public static final GTMaterial Zinc = new GTMaterial(30,"Zinc", 1, 250, 240, 240, false, molten, smalldust, dust, nugget, ingot, plate, stick, gear, blockMetal, tinydust);
```
Comment (Trinsdar) [inline src/main/java/gtc_expansion/material/GTCXMaterial.java]: Just curious: why not smeltable?

Comment (DbpGaming) [reply src/main/java/gtc_expansion/material/GTCXMaterial.java]: oh shit, that shouldve been on cassiterite, dont know how i fucked that up

File: src/main/java/gtc_expansion/tile/multi/GTCXTileMultiIndustrialBlastFurnace.java (hunk 1 above)
Comment (DbpGaming) [reply src/main/java/gtc_expansion/tile/multi/GTCXTileMultiIndustrialBlastFurnace.java]: mm okay, 1000 just allows you to skip the steel IBF when making aluminium, thought it was unintentional, will revert it

**Comment by Trinsdar** [comment]
Ok i can't see anything else that needs to be changed, is it ready to merge then?

**Comment by DbpGaming** [comment]
yeah theres more changes i want to do, but most of them are compat stuff, so they will be in separate pr's and done later

**Pull request merged**